src/corner_kick/src/components/Portal/README.md

```diff
@@ -0,0 +1,50 @@
+# Portals - Design notes<!-- omit in toc -->
```

Comment:
Could we put this in main `corner_kick` README? I know it's pretty long, but in my experience there's a strong negative relationship between how deep markdown files are in the file tree and how often they get read, and this has some really important information that people should definitely understand when working on the visualizer.

Reply:
The benefit of this approach is that the README is closer to where the implementation is. I don't agree with adding to much information in the main README because this is not a sustainable approach. Perhaps an alternative is to keep a wiki in a dedicated folder with all design notes.

Reply:
I see what you mean. How're we going to counteract people not looking in the "Portals" folder for documentation? I don't think a separate folder is the right direction here (because as you mentioned, we want to keep this close to the code), and this definitely doesn't belong on the wiki.

Reply:
I would prefer not to. I will have more design docs coming along, as other areas of the application get defined. In the near future, I expect docs to come up for the sidebar, shape system, layer system, redux, ros interface, etc. I want to make those docs as thorough as possible and keep them as close to the codebase as possible

Reply:
A large README will not help with readability in this case

Reply:
I get where you're coming from here. Ideally we don't have spread out documention that's not at least indexed, so would it not be reasonable to put this in the README for now, and then put splitting it out under #244 so that having spread out documentation and having it all indexed are coupled?